scarb/src/bin/scarb/commands/fetch.rs

```diff
@@ -0,0 +1,14 @@
+use anyhow::Result;
+
+use scarb::core::Config;
+use scarb::ops;
+
+#[tracing::instrument(skip_all, level = "info")]
+pub fn run(config: &Config) -> Result<()> {
+    let ws = ops::read_workspace(config.manifest_path(), config)?;
+
+    match ops::resolve_workspace(&ws) {
+        Ok(_) => Ok(()),
+        Err(e) => Err(e),
+    }
```

Comment:
protip, no need to change:

```suggestion
    ops::resolve_workspace(&ws).map(|_| ())
```